chromadb/utils/embedding_functions.py

```diff
@@ -801,6 +801,35 @@ def __call__(self, input: Documents) -> Embeddings:
             self._api_url, json={"inputs": input}
         ).json()
 
+class TogetherEmbeddingFunction(EmbeddingFunction[Documents]):
+    """
+    This class is used to get embeddings for a list of texts from together's embedding models.
+    It requires an API key and a model name. The default model name is "togethercomputer/m2-bert-80M-8k-retrieval".
+    """
+    def __init__(self, api_key: str, model_name: str = "togethercomputer/m2-bert-80M-8k-retrieval"):
```

Comment:
params should accept an API to match other embedding_functions

Reply:
I couldn't get it. How would it accept an API? I think this is similar to other functions. @jeffchuber 
https://github.com/chroma-core/chroma/blob/a669624deeb9a5a2089b61303f6685159318d30f/chromadb/utils/embedding_functions.py#L215